Docker: Use a volume for the database storage

ryelle opened the pull request:
This is transparently managed by docker, not listed as a directory in the host filesystem. We don't need to expose the mysql internals, and this is still persistent. MySQL is still reachable by using the docker CLI.

_This is one of the changes I made when trying to get my local env working again, and while I'm not sure it's related, it does clean up the host filesystem._

⚠️  I imagine switching to this will probably blow away the existing database, so export or backup anything you want before doing so.

ryelle committed:
Docker: Use a volume for the database storage

This is transparently managed by docker, not listed as a directory in the host filesystem. We don't need to expose the mysql internals, and this is still persistent. MySQL is still reachable by using the docker CLI.

iandunn reviewed (commented):
Seems fine at first glance, but I don't use the Docker env either, so no preference.

ryelle committed:
Blocks: Switch away from deprecated hook name

ryelle committed:
Docker: Install php intl library required by CampTix

Without this, the NumberFormatter class is not found, which causes a fatal error in the ticket flow.

ryelle committed:
Update volume name, add network

ryelle commented:
Closing this, I've stopped using this method, and I don't think it had anything to do with whatever my issue was in the first place.
